packages/core/helper-plugin/lib/src/hooks/useAPIErrorHandler/utils/getPrefixedId/index.js

```diff
@@ -0,0 +1,14 @@
+const ERROR_PREFIX = 'apiError.';
+
+export function getPrefixedId(message, callback) {
+  const prefixedMessage = `${ERROR_PREFIX}${message}`;
+
+  // if a prefix function has been passed in it is used to
+  // prefix the id, e.g. to allow an error message to be
+  // set only for a localization namespace
+  if (callback) {
```

Comment:
should we do `if(typeof callback === "function")` because we don't have typesafety and this would be a public API

Reply:
Done in 17f773ebd390eb72fd32cf434eeaad250be7abab